src/backend/.gitignore

```
@@ -0,0 +1 @@
+ralph_auth.json
```

I would prefer the following in the main project `.gitignore` file:

```suggestion
.ralph/auth.json
```